src/annotator/anchoring/match-quote.ts

```diff
@@ -1,31 +1,27 @@
 import approxSearch from 'approx-string-match';
+import type { Match as StringMatch } from 'approx-string-match';
 
-/**
- * @typedef {import('approx-string-match').Match} StringMatch
- */
-
-/**
- * @typedef Match
- * @prop {number} start - Start offset of match in text
- * @prop {number} end - End offset of match in text
- * @prop {number} score -
- *   Score for the match between 0 and 1.0, where 1.0 indicates a perfect match
- *   for the quote and context.
- */
+type Match = {
+  /** Start offset of match in text */
+  start: number;
+  /** End offset of match in text */
+  end: number;
+  /**
```

Comment:
World's most pedantic PR comment, and one you can take or leave: I tend to leave an extra line break above multi-line property comments. So:

```
type Whatever = {
  /** Short comment about this property **/
  foo: string;
  /** Short comment about this property **/
  bar: string;

  /**
   * A multi line...
   * ...
   * comment about this property
   */
  baz: number;
}
```;

Reply:
Oh, I wouldn't expect any of our code to actually be _consistent_ in this respect, hah!